api/_bot/dev.js

```diff
@@ -1,3 +1,4 @@
+import 'regenerator-runtime'
```

Comment:
useful when testing async methods/funcs

Reply:
changed this by using a custom configuration of babel